src/Collections/OrganizationSiteMemberships.php

```diff
@@ -82,9 +83,6 @@ public function get($id)
      */
     public function getOrganization()
     {
-        if (empty($this->organization)) {
-            $this->organization = $this->getContainer()->get(Organization::class, [$this->get('organization')]);
```

Comment:
I was never fully satisfied with the way getting/setting the 'parent' object of a collection or model works. It's a bit inconsistent and not very intuitive. But it works.

Does removing this code not cause problems though? How is the org set?